system/include/webgpu/webgpu_cpp.h

```diff
@@ -1236,6 +1257,7 @@ namespace wgpu {
         char const * driverDescription;
```

Comment:
system/include/webgpu/webgpu_cpp.h line 1245
The ChainedStruct and ChainedStructOut is missing from the dawn gen emscripten_bits files. But looks like they should exist here. Likely a side effect of https://dawn-review.googlesource.com/c/dawn/+/132268.
If we don't want to keep a new separate `webgpu_cpp_chained_struct.h`, maybe just keep it like the other bitmask thing when updating.